config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-1-2.yaml

```diff
@@ -134,7 +134,7 @@ periodics:
       - name: INIT_WITH_PROVIDERS_CONTRACT
         value: v1beta1
       - name: INIT_WITH_KUBERNETES_VERSION
-        value: v1.25.0
+        value: v1.26.0
```

Comment:
This will start a CAPI v1.1.5 cluster with Kubernetes version v1.26 which is not part of our support matrix. I realise that we did this for the previous kubernetes versions but just curious if we should continue doing this?

Reply:
I think it's okay, given that it keeps the job in test-infra in sync with the code inside of the branch (we will bump this in the release-1.2 branch when we cherry-pick the 1.26 change into release-1.2).

This will be the last time we have to do this in test-infra as the additional upgrade jobs have been moved into clusterctl_upgrade_test.go

Reply:
Just to clarify. It will create a v1.26 cluster with Cluster API v1.2.x and then it will deploy Cluster API v1.1.5 on top of that v1.26 cluster. It won't create a v1.26 cluster with Cluster API v1.1.x

(still outside of the support matrix to run Cluster API v1.1.x on top of v1.26, but just wanted to make sure we're talking about the same thing)

Reply:
So I should be reverting back to 1.25?

Reply:
Thought about it some more. Yuvaraj is right. Let's use 1.24 here. It's the latest version supported by 1.1

Reply:
right